.github/website-challenge-md126/yonisE.css

```
@@ -0,0 +1,171 @@
+.blocks {
+    width: 90%;
+    max-width: 1000px;
+    min-height: 2em;
+    height: auto;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr; 
+    grid-template-rows: auto;
+    grid-template-areas:
+    "header header header"
+    "profile pic about"
+    "game game game"
+    "water dream scratch"
+    "footer footer footer"
+}
+
+body {
+    background:whitesmoke);
+}
+
+*{font-family: 'Courier New', Courier, monospace;}
```

Look into a linter such as Prettier which will format code automatically on save. Formatted code is best practice, code should be consistent in positioning and placement to improve readability. Make sure you have Prettier installed on your VSCode. 
```suggestion
* {
    font-family: 'Courier New', Courier, monospace;
}
```